home-open-api/src/main/resources/yaml/openapi.yaml

```diff
@@ -1627,3 +2035,244 @@ components:
+        - inactive
+    ReadOwnership:
+      allOf:
+        - $ref: '#/components/schemas/BaseReadView'
+      type: object
+      properties:
+        user:
+          $ref: '#/components/schemas/ReadUser'
+        ownership_part:
+          type: number
+          minimum: 0.0001
+          maximum: 1.0
+          multipleOf: 1e-4
+          example: 0.6588
+        status:
+          $ref: '#/components/schemas/OwnershipStatus'
+
+    CreateApartment:
+      required:
+        - number
+        - area
+      type: object
+      properties:
+        number:
+          type: string
+          minLength: 1
+          maxLength: 6
+          pattern: '^(?!(0))\d{1,4}(\-[a-zа-я])?$'
+          example: 15
+        area:
+          type: number
+          minimum: 10
+          maximum: 1000
+          multipleOf: 1e-2
+          example: 75.28
+        invitations:
+          type: array
+          items:
+            $ref: '#/components/schemas/CreateApartmentInvitation'
+          example:
+            - email: other1@gmail.com
+              ownership_area: 30.0
+            - email: other2@gmail.com
+              ownership_area: 42.3
+    UpdateApartment:
+      required:
+        - number
+        - area
+      type: object
+      properties:
+        number:
+          type: string
+          minLength: 1
+          maxLength: 6
+          pattern: '^(?!(0))\d{1,4}(\-[a-zа-я])?$'
+          example: 15
+        area:
+          type: number
+          minimum: 10
+          maximum: 1000
+          multipleOf: 1e-2
+          example: 75.28
+    ReadApartment:
+      allOf:
+        - $ref: '#/components/schemas/BaseReadView'
+      type: object
+      properties:
+        number:
+          type: string
+          minLength: 1
+          maxLength: 6
+          pattern: '^(?!(0))\d{1,4}(\-[a-zа-я])?$'
+          example: 15
+        area:
+          type: number
+          minimum: 10
+          maximum: 1000
+          multipleOf: 1e-2
+          example: 75.28
+        ownerships:
+          type: array
+          items:
+            $ref: '#/components/schemas/ReadOwnership'
+          example:
+            - id: 1
+              user:
+                id: 1
+                first_name: Petro
+                last_name: Petrov
+                contacts:
+                  - id: 1
+                    type: email
+                    main: true
+                  - id: 2
+                    type: phone
+                    main: false
+              ownership_part: 0.72
+              status: active
+            - id: 2
+              user:
+                id: 2
+                first_name: Ivan
+                last_name: Ivanov
+                contacts:
+                  - id: 3
+                    type: email
+                    main: true
+                  - id: 4
+                    type: phone
+                    main: false
+              ownership_part: 0.38
+              status: active
+        invitations:
```

Comment:
Let's do not expose invitations as part of `ReadApartment`. We will use a sub-endpoint for this.

Reply:
> Let's do not expose invitations as part of `ReadApartment`. We will use a sub-endpoint for this.

Proposed changes have been made